custom_ops/gpu_ops/moe/tritonmoe_preprocess.cu

```diff
@@ -47,8 +47,12 @@ __global__ void moe_align_block_size_kernel(const scalar_t* __restrict__ topk_id
 
   __syncthreads();
 
-  for (int i = threadIdx.x; i < numel; i += blockDim.x) {
+  size_t block_size = numel / 1024;
```

Comment:
Confused Why using `size_t block_size = numel / 1024;`?
Usually we use `for (int i = threadIdx.x; i < numel; i += blockDim.x) {` to deal numel elements array.